src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -1,60 +1,83 @@
 package mate.academy.model;
 
 import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
 import jakarta.persistence.JoinColumn;
 import jakarta.persistence.JoinTable;
-import jakarta.persistence.MapsId;
 import jakarta.persistence.OneToMany;
 import jakarta.persistence.OneToOne;
 import jakarta.persistence.Table;
 import java.util.List;
+import java.util.Objects;
 
 @Entity
-@Table (name = "shopping_carts")
+@Table(name = "ShoppingCarts")
 public class ShoppingCart {
     @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @OneToMany
-    @JoinTable(name = "shopping_carts_tickets",
+    @JoinTable(name = "shopping_cart_contents",
             joinColumns = @JoinColumn(name = "shopping_cart_id"),
             inverseJoinColumns = @JoinColumn(name = "ticket_id"))
     private List<Ticket> tickets;
     @OneToOne
```

Comment:
The `@OneToOne` annotation for the `user` field should include a `@JoinColumn` to specify the foreign key column name in the database. Consider adding `@JoinColumn(name = "user_id")` to clarify the relationship.